Rasa_Bot/requirements.txt

```diff
@@ -1,2 +1,5 @@
 -r actions/requirements-actions.txt
-rasa==2.8.0
\ No newline at end of file
+rasa==2.8.0 # NB! when updating, make sure to also update:
+            # * rasa-sdk base image in Dockerfile
+            # * format of stories, domain.yml etc. if necessary
```

Comment:
No need to update requirements in root folder right? Because it imports these requirements.